apps/api/openapi/openapi.yaml

```diff
@@ -1152,6 +1181,110 @@ components:
       schema:
         type: integer
         example: 10
+    Title:
+      name: title
+      in: query
+      description: "Filtre: Le titre de l'offre d'emploi. Opérateur par défaut: %l%"
+      required: false
+      schema:
+        type: string
+        example: 'Lead'
+    Skills:
+      name: skills
```

Comment:
Tiens ! Ca y'est je me souvient du vrai argument sur le ou mettre les operateurs ! Sur le nom du filtre ou sur la valeur ... 
Et bien par exemple, dans le cas d'OpenAPI, il vaut mieux en fait, j'ai l'impression, qu'il soit au niveau de la valeur. Par ce que si on mettait un filtre strict de validation du contrat OpenAPI, on accepterait `skills`, mais pas `skills:eq` ou `skills:in` ... Par contre, sur la valeur, et bien cela reste une string ....
Qu'en penses-tu ?

Reply:
J'en pense que c'est malin ! Enfin j'avais tout réécrit pour la syntaxe `key(:operator)=value`, je peux reswitcher à `key=value(:operator)`, ça va vite. Et ça rends swagger plus utilisable. C'est que tu avais parlé de cet opérateur `:in`...

Reply:
Oui, et bien c'est une bonne leçon d'apprise ! Je dis tout plein de truc pas malin ^^

Reply:
OK je m'y mets.

Reply:
done !